firmware/lib/thermistorProbe/thermistorUSP10976.h

```diff
@@ -0,0 +1,44 @@
+#pragma once
+#ifndef THERMISTOR_USP10976_H
+#define THERMISTOR_USP10976_H
+
+class Thermistor_10k
```

Comment:
Can we name the file the same as the class it contains. Also would prefer no underscores in class names, just CamelCase